integration-tests-2/test_base_llms.py

```diff
@@ -0,0 +1,47 @@
+import requests
+from utils.docker_runner import run_lorax_container
+
+
+def test_base_mistral():
```

Comment:
This will be fun to make data-driven using `pytest.mark.parametrize` in a follow-up :)

But good for v1.